src/test/java/com/inventorsoft/junit/mapper/BookMapperTest.java

```diff
@@ -0,0 +1,185 @@
+package com.inventorsoft.junit.mapper;
+
+import com.inventorsoft.junit.dto.request.CreateBookRequest;
+import com.inventorsoft.junit.dto.request.UpdateBookRequest;
+import com.inventorsoft.junit.dto.response.AuthorResponse;
+import com.inventorsoft.junit.dto.response.BookResponse;
+import com.inventorsoft.junit.model.Author;
+import com.inventorsoft.junit.model.Book;
+import com.inventorsoft.junit.repository.AuthorRepository;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.IntStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@FieldDefaults(level = AccessLevel.PRIVATE)
+class BookMapperTest {
+
+    @Mock AuthorRepository authorRepository;
+    @Mock AuthorMapper authorMapper;
+    @InjectMocks BookMapper bookMapper;
+
+    static final long AUTHOR_ID = 35L;
+    static final String AUTHOR_NAME = "Leo Tolstoy";
+    static final LocalDate AUTHOR_BIRTHDAY = LocalDate.of(1828, 9, 9);
+
+    static final long BOOK_ID = 5L;
+    static final String BOOK_TITLE = "War and Peace";
+    static final LocalDate BOOK_RELEASE_DATE = LocalDate.of(1869, 5, 23);
+    static final String BOOK_DESCRIPTION = """
+             War and Peace broadly focuses on Napoleon’s invasion of Russia in 1812 and 
+             follows three of the most well-known characters in literature: Pierre Bezukhov, the illegitimate 
+             son of a count who is fighting for his inheritance and yearning for spiritual fulfillment;
+             Prince Andrei Bolkonsky, who leaves his family behind to fight in the war against Napoleon; 
+             and Natasha Rostov, the beautiful young daughter of a nobleman who intrigues both men.
+            """;
```

Comment:
basically, in unit tests there is not need to have real-word data, some dummy text also good